crates/bevy_pbr/src/render/pbr_lighting.wgsl

```diff
@@ -131,14 +135,19 @@ fn Fd_Burley(roughness: f32, NoV: f32, NoL: f32, LoH: f32) -> f32 {
     return lightScatter * viewScatter * (1.0 / PI);
 }
 
-// From https://www.unrealengine.com/en-US/blog/physically-based-shading-on-mobile
-fn EnvBRDFApprox(f0: vec3<f32>, perceptual_roughness: f32, NoV: f32) -> vec3<f32> {
+// Scale/bias approximation
+// https://www.unrealengine.com/en-US/blog/physically-based-shading-on-mobile
+// TODO: Use a LUT (more accurate)
```

Comment:
Presumably we might encounter issues like this until we add this?
https://github.com/google/filament/discussions/4989

Reply:
Not sure. Haven't seen anything in testing though.